pkgs/by-name/li/libansilove/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gd
+,
+}:
```

Comment:
Nixpkgs no longer uses this style of formatting - please reformat this file using `nixfmt-rfc-style`, which produces lists like this:
```nix
{
  lib,
  stdenv,
  // and so on
}
```